src/Perspex.Controls/Window.cs

```diff
@@ -44,6 +47,22 @@ public enum SizeToContent
     /// </summary>
     public class Window : TopLevel, IStyleable, IFocusScope, ILayoutRoot, INameScope
     {
+        private static IList<Window> windows = new List<Window>();
+
+        /// <summary>
+        /// Retrieves an enumeration of all Windows in the currently running application.
+        /// Can only be accessed from the UI Thread.
+        /// </summary>
+        public static IEnumerable<Window> Windows
+        {
+            get
+            {
+                Dispatcher.UIThread.VerifyAccess();
```

Comment:
It seems a kinda weird to have a `VerifyAccess` check on a property getter - these are usually put on methods that change state.